geom/tests/geom_setup.F90

```diff
@@ -0,0 +1,162 @@
+#include "MAPL_Generic.h"
+
+module geom_setup
+
+   use ESMF
+   use funit
+   use MAPL_ExceptionHandling
+
```

Comment:
Note to self:  in long term we may want to have a separate directory of test utilities that can be used in multiple MAPL directories.   These would create simple grids, fields, etc. for other tests to use.       Otherwise we get some duplication with varying quality for these sorts of things.

Reply:
Absolutely agree 100%